README.md

```diff
@@ -75,10 +76,10 @@ before opening a pull request.
 
 Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved.
 This file is licensed under the Apache Software License, v. 2 except as noted
-otherwise in the [LICENSE file][license].
+otherwise in the [LICENSE file](https://github.com/SAP/devops-docker-images/blob/master/LICENSE).
```

Comment:
You fixed the alias below -- is the change in this line here still required?

Reply:
I was not aware of this alias feature, so I guess you're right. Thanks, will fix it.